Sources/SwiftFortuneWheel/SwiftFortuneWheel.swift

```diff
@@ -415,7 +415,9 @@ public extension SwiftFortuneWheel {
         self.animator.addRotationAnimation(fullRotationsCount: 0,
                                            animationDuration: animationDuration,
                                            rotationOffset: rotation,
-                                           completionBlock: nil)
+                                           completionBlock: { _ in
+            self.pauseAllSound()
```

Comment:
Calling pauseAllSound after the rotation animation stops is a bad idea. Same audio effects could have more than 1-second duration, and stopping all sound will cut the sound instantly, making sound playback unnatural.

Reply:
I'll come up with other suggestion if there is any from my side